build.assets/robotest_run_nightly.sh

```diff
@@ -5,11 +5,22 @@ readonly UPGRADE_FROM_DIR=${1:-$(pwd)/../upgrade_from}
 
 DOCKER_STORAGE_DRIVERS="overlay2"
 
+# UPGRADE_MAP maps gravity version -> list of OS releases to upgrade from
 declare -A UPGRADE_MAP
-# gravity version -> list of OS releases to test upgrades on
-UPGRADE_MAP[5.2.15]="centos:7 ubuntu:16"
+# latest patch release on compatible LTS, keep these up to date
+UPGRADE_MAP[5.5.37]="centos:7 ubuntu:16"
```

Comment:
Why thing that was popped up a while ago is keeping these versions in sync. I think if we moved them to the makefile closer to where the versions are defined / queried, it would be much easier to handle.
Something along the lines of:
```
VERSIONS_LTS_TO_LATEST = 5.5.37 5.2.15
VERSIONS_TO_LATEST = 5.2.12 5.5.19 5.5.20 5.5.28 5.5.36
```
Also, we need to set up nightly if not yet done for `5.5.x`.

Reply:
> Also, we need to set up nightly if not yet done for 5.5.x.

Excellent point.  This does not yet exist.  I'll set it up.

> One thing that popped up a while ago is keeping these versions in sync. I think if we moved them to the makefile closer to where the versions are defined / queried, it would be much easier to handle.
> Something along the lines of:
> 
> ```
> VERSIONS_LTS_TO_LATEST = 5.5.37 5.2.15
> VERSIONS_TO_LATEST = 5.2.12 5.5.19 5.5.20 5.5.28 5.5.36
> ```

I've filed #1199 to track this work.